doc/classes/NavigationServer2D.xml

```diff
@@ -457,7 +457,7 @@
 				Create a new map.
 			</description>
 		</method>
-		<method name="map_force_update">
+		<method name="map_force_update" deprecated="No longer supported as incompatible with async updates. Can only be used in single-threaded context at own risk.">
```

Comment:
```suggestion
		<method name="map_force_update" deprecated="This method is no longer supported, as it's incompatible with asynchronous updates. It can only be used in a single-threaded context, at your own risk.">
```
Same for the other one.